docs/sentry/react-native.md

```diff
@@ -0,0 +1,84 @@
+# Sentry React Native
+
+
+### Getting Started
+Install all needed dependencies:
+* [@sentry/react-native](https://www.npmjs.com/package/@sentry/react-native)
+
+```bash
+npm i @sentry/react-native
+#or
+yarn add @sentry/react-native
+```
+
+Run the Sentry wizard to automatically patch native files. (Hint: The wizard doesn't support React Native versions 0.68 and older)  
+**Remember to work on a clean workspace and commit current changes. So in case of failure you don't mess up your code.**
+```bash
+npx @sentry/wizard@latest -s -i reactNative
+```
+
+The wizard creates two sentry.properties files. One in the android and one in the ios native directories.  
+Add `sentry.properties` to `.gitignore` as it contains the secret access token for your CI.  
+You can copy the contents of this file as base64 and add it as a secret in your workflow.
+```bash
+# This copies the file to your clipboard
+cat ios/sentry.properties | openssl base64 | tr -d '\n' | pbcopy
+```
+Add the base64 string as a secret to your github repository like `SENTRY_PROPERTIES_FILE_BASE64` with the content of your clipboard  
```

Comment:
I asked this myself lately? Do we actually need base64 here? I have a private repo where the file content is pasted in directly. Works as well. Since base64 can be decoded at any time there is no layer of security here. So why did we start with encoding this as base64 anyway? :D

Reply:
Hm yeah we dont really need it :D 
I thought that if you have a large file you have only one string - but if I think about it again it makes no sense as we can just copy paste the file content in the secret field. In on of my project I also just copy the env file raw in the secret field and it works as well